samples/BikeSharingApp/ReservationEngine/TESTING.md

```diff
@@ -0,0 +1,8 @@
+### Local Debugging pre-requisites
+* ```dotnet restore```
+* [Debugging dotnet core app in VS Code](https://docs.microsoft.com/en-us/dotnet/core/tutorials/debugging-with-visual-studio-code)
+
+<br/>
+
+### Additional information
+* Service port : 49308
```

Comment:
>49308 [](start = 17, length = 5)

I believe this one should be "5000", as we just take the default .NET Core port for the ReservationEngine. I think this port is used by IIS, but it's not the one we listen to locally by default.

Reply:
I think you're right... updated